extralibs/fsusb.c

```diff
@@ -417,6 +417,8 @@ void USBFS_IRQHandler()
 						if( len > USBFS_SetupReqLen )
 							len = USBFS_SetupReqLen;
 						ctx->USBFS_SetupReqLen = len;
+
+						break;
```

Comment:
I'm 99% sure this will break a lot of things, this relies on the code falling through from above. You will need to find some other way with fallthrough to inform GCC, like `__attribute__ ((fallthrough));`